PRIVACY_AND_SECURITY.md

```diff
@@ -0,0 +1,82 @@
+# Answers to [Security and Privacy Questionnaire](https://www.w3.org/TR/security-privacy-questionnaire/)
+
+### 3.1 Does this specification deal with personally-identifiable information?
+
+No.
+
+
+### 3.2 Does this specification deal with high-value data?
+
+No.
+
+
+### 3.3 Does this specification introduce new state for an origin that persists across browsing sessions?
+
+No.
+
+
+### 3.4 Does this specification expose persistent, cross-origin state to the web?
+
+No.
+
+### 3.5 Does this specification expose any other data to an origin that it doesn’t currently have access to?
+
+Kind of. Origins will explicitly be told how they have been launched. Previously, they would have had to infer this based on requests (though this information **WAS** previously inferrible).
+
+### 3.6 Does this specification enable new script execution/loading mechanisms?
+
+No.
+
+
+### 3.7 Does this specification allow an origin access to a user’s location?
+
+No.
+
+
+### 3.8 Does this specification allow an origin access to sensors on a user’s device?
+
+No.
+
+
+### 3.9 Does this specification allow an origin access to aspects of a user’s local computing environment?
+
+No.
+
+
+### 3.10 Does this specification allow an origin access to other devices?
+
+No.
+
+
+### 3.11 Does this specification allow an origin some measure of control over a user agent’s native UI?
+
+No.
+
+
+### 3.12 Does this specification expose temporary identifiers to the web?
+
+No.
+
+
+### 3.13 Does this specification distinguish between behavior in first-party and third-party contexts?
+
+Only first party contexts will receive launch events.
+
+
+### 3.14 How should this specification work in the context of a user agent’s "incognito" mode?
+
+Launch events will be gated by installation. Sites may not be installed in incognito mode, so, while a site can still specify a launch event handler, it will never be fired (nor can they expect it to, even in non-incognito modes).
```

Comment:
If we didn't gate on installation would we still fire launch events in incognito mode? I don't see any reason not to. Perhaps we should just say that it should work the same between incognito and non-incognito? (Though it would be profile isolated - as in clicks in a non-incognito tab wouldn't be captured by an app running in an incognito tab)

Reply:
:+1: